README.md

```
@@ -10,14 +10,35 @@ All version numbers are controlled by `VERSION` at root level, no need to change
 
 For things work properly , be sure to follow build methods documented in [Build.md](Build.md)
 
-## Emulator support
+## Launch Parameters
+
+### HTTP Server Port Configuration
+
+OneKey bridge runs an HTTP server on port 21320 by default. You can change this port using the `-p` parameter.
+
+**Examples:**
+- `./onekeyd` - Uses default port 21320
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)